front/lib/resources/data_source_view_resource.ts

```diff
@@ -249,7 +250,11 @@ export class DataSourceViewResource extends ResourceWithSpace<DataSourceViewMode
       }
     );
 
-    return dataSourceViews.filter((dsv) => dsv.canList(auth));
+    return dataSourceViews.filter(
+      (dsv) =>
+        (!dsv.space.isConversations() || includeConversationDataSources) &&
```

Comment:
In case you are wondering, space is fetched already by default (and used in `canRead()`)